config/locales/en.yml

```diff
@@ -1300,11 +1308,13 @@ en:
       subject:
         disable: Your account %{acct} has been frozen
         none: Warning for %{acct}
+        sensitive: Your account %{acct} marked as sensive
         silence: Your account %{acct} has been limited
         suspend: Your account %{acct} has been suspended
       title:
         disable: Account frozen
         none: Warning
+        sensitive: Account is marked as sensive
```

Comment:
Typo: `sensive` -> `sensitive`

I am a little concerned about the choice of words. This is consistent with "Mark media as sensitive" but "sensitive" has a different meaning when talking about people: "easily hurt or damaged"

Reply:
Oops!

How about this notation?
> Mark the media and URLs that your account posts as sensitive